packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -451,6 +451,7 @@ class FlutterRunTestDriver extends FlutterTestDriver {
     bool singleWidgetReloads = false,
     File pidFile,
     String script,
+    List<String> args,
```

Comment:
intentional or test code? Please add comment for args otherwise

Reply:
Its all test code, but i changed the name to be more descriptive.